Topics/Development_Process/Build_Requirements/CMakeLists_txt.md

```diff
@@ -0,0 +1,199 @@
+# **Learning CMakeLists.txt**
+
+## Table of Contents
+### [1. Brief Introduction to CMake](#brief-introduction-to-cmake)
+### [2. What is CMakeLists.txt?](#what-is-cmakeliststxt)
+### [3. Creating CMakeLists.txt](#creating-cmakeliststxt)
+### [4. Other Notes](#other-notes)
+### [5. References](#references)
+
+---
+## Brief Introduction to CMake
+
+This article will help readers understand what `CMakeLists.txt` is, why it is used, and provide instructions on how to create a `CMakeLists.txt`. Before getting started, it is important to understand what is CMake. Note that it is presumed that the reader should have the very basic knowledge of C++, Make, and MakeFile.
```

Comment:
Grammar:
- "what is CMake" -> "what CMake is"
- The final sentence can probably be reworded in a more straightforward way -- perhaps something like "Note that this article assumes very basic knowledge of [...]"?